src/icons/ArrowUp.tsx

```diff
@@ -0,0 +1,36 @@
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+const pathClassName = classnames(
+  stroke('stroke-primary-light'),
+  strokeWidth('stroke-4')
+)
+
+export default function ({ small }: { small?: boolean }) {
+  if (small)
```

Comment:
You don't need 2 svgs, you need 2 different `div` containers, your `svg` should take whole place in those:
```TS
if (small) return <div className="w-19 h-27><SimpArrow /></div>

return <div className="w-27 h-40"><SimpArrow /></div>
```

Reply:
They are a little bit different with different paths. So it's better to keep both versions